Makefile

```diff
@@ -0,0 +1,12 @@
+GOFMT=goimports
+
+goimports:
+	${GOFMT} -w ./
+
+goimports-check:
+	$(eval diff_files = $(shell ${GOFMT} -l ./))
+	@if [ -n "${diff_files}" ]; then \
+		echo "Please run 'make goimports' to fix the format errors in the files:"; \
+		echo "\t${diff_files}"; \
+		exit 1; \
+	fi;
```

Comment:
I recommend to check this via golangci-lint, like here:
https://github.com/cri-o/cri-o/blob/2dc77fc03c522237a353ca8dfbd9a69c1319bed6/Makefile#L236-L241